src/app/lib/config/services/igbo.js

```diff
@@ -15,6 +15,7 @@ const igbo = {
   brandSVG,
   script: latin,
   manifestPath: '/manifest.json',
+  serviceWorkerPath: '/igbo/sw.js',
```

Comment:
I prefer shortening `serviceWorkerPath` to something like `swPath` since the name service in serviceWorkerPath does not have anything to do with BBC services

Reply:
Thats fair, idm what its called, just that its value is consistent with that of manifestPath :)
'/igbo/sw.js' & '/igbo/manifest.json'
or
'/sw.js' & '/manifest.json'

Reply:
^ I dont think this has been done 🙈